### PR TITLE
index.ts should be the only toplevel src file

### PR DESCRIPTION
**Pull request by rictic**
- [x] internal change, CHANGELOG.md not updated

### CODE REVIEW

File: src/test/core/dependency-graph_test.ts [hunk 1]
```diff
@@ -17,9 +17,9 @@
 import {assert, use} from 'chai';
 import * as path from 'path';
 
-import {Analyzer} from '../../analyzer';
+import {Analyzer} from '../../core/analyzer';
 import {DependencyGraph} from '../../core/dependency-graph';
-import {FSUrlLoader} from '../../url-loader/fs-url-loader';
+import {FSUrlLoader} from '../../index';
```
Comment (justinfagnani) [inline src/test/core/dependency-graph_test.ts]: I don't think we should import from index.ts internally, even in tests

**Comment by justinfagnani** [comment]
meh. Honestly, I really dislike `index.js` files. A package does not need to appear like a single module, and it's not actually an "index" of anything. but ok.

**Comment by rictic** [comment]
I agree that a package doesn't need to look like a single module, but I do like having a clear delineation between public API and private implementation details.

File: src/test/core/dependency-graph_test.ts (hunk 1 above)
Comment (rictic) [inline src/test/core/dependency-graph_test.ts]: Done, here and elsewhere.

**Pull request merged**